content/en/platform/corda/5.0/developing-applications/getting-started/first-flow/code-kotlin.md

```diff
@@ -337,7 +337,7 @@ We can now start sending messages to the responder:
 ## Other Considerations for FlowSessions
 It is important that the sends and receives in the initiator and responder flows match. If the initiator sends a Foo and the responder expects a Bar, the flow hangs and likely results in a timeout error.
 
-As with Corda 4, there is also a `sendAndReceive` method on `FlowSession` that sends a payload, check-points the flow, and then waits for a response to be received:
+The `sendAndReceive` method on `FlowSession` sends a payload, check-points the flow, and then waits for a response to be received:
 ```kotlin
 val response = myFlowSession.sendAndReceive(<ReceiveType>::class.java, payload)
 ```
```

Comment:
We might do the same changes in the java version of this page.

Reply:
Updated java version of the page.